activation/activation.go

```diff
@@ -176,6 +177,11 @@ func (b *Builder) StartSmeshing(coinbase types.Address, opts PostSetupOpts) erro
 		return errors.New("already started")
 	}
 
+	err := b.postSetupProvider.PrepareInitializer(b.parentCtx, opts)
```

Comment:
Hi @dshulyak we could try to extract side effect-free validation but whatever we extract it seem's like it won't catch as many problems as just calling `PrepareInitializer` because `PrepareInitializer` is doing the actual work.

Also doing the extraction will take longer and @pigmej mentioned that he'd like this fix "fast"

So yes it's possible to do the extraction, but I'm not sure it would provide any benefit.

Also what do you mean by "_temporarily_ panic"?

Reply:
My thinking is that if I want smeshing and I cannot, then we literally should quit asap because otherwise the user may have false assumptions. 

Obviously if it can be done in a pretty way then we can :) Just having an ugly fix now is better than not at all IMHO.

Reply:
sounds good, if this approach doesn't cause other issues. for example calling PrepareInitializer several times and similar simple things

Reply:
My current understanding is that `PrepareInitializer` sets some state in the `PostSetupManager` and calling it again simply overwrites that state. So I don't think multiple calls to it will pose a problem.

Reply:
But I realised there is now a race condition with `StartSession`, i need to think about this a bit more